bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_tau_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_tau_LO/HeavyNeutrino_trilepton_V-0p01_tau_LO_run_card.dat

```diff
@@ -0,0 +1,259 @@
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	=  run_tag  ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false. =  gridpack   !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000	=  nevents  ! Number of unweighted events requested 
+  0	=  iseed    ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+  1	=  lpp1     ! beam 1 type 
+  1	=  lpp2     ! beam 2 type
+  6500.0	=  ebeam1   ! beam 1 total energy in GeV
+  6500.0	=  ebeam2   ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+  0.0	=  polbeam1  ! beam polarization for beam 1
+  0.0	=  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf        =  pdlabel    ! PDF set
+  262000        =  lhaid      ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
As above. Make sure variations are in runcmsgrid.sh.

Reply:
Thanks to Sihyun I confirm the variations are in runcmsgrid.sh